SignalMessaging/categories/Theme.m

```diff
@@ -49,6 +54,12 @@ + (UIColor *)backgroundColor
     return (Theme.isDarkThemeEnabled ? UIColor.ows_blackColor : UIColor.ows_whiteColor);
 }
 
++ (UIColor *)offBackgroundColor
+{
+    return (
+        Theme.isDarkThemeEnabled ? [UIColor colorWithWhite:0.2f alpha:1.f] : [UIColor colorWithWhite:0.9f alpha:1.f]);
```

Comment:
I'm sure some of this was just following the legacy pattern, but here and elsewhere, would it be better to specify opaque grays rather than translucent blacks so we don't:

1.  inadvertently show content behind these colors
2. I believe there are perf implications to using translucent colors. Not a big deal on simple VC's, but there could be wins in hotspots like our collection views.

Reply:
I think this comment might be misplaced?  These colors _are_ opaque.  

There are very few transparent colors in this PR and all of them a) were already transparent b) appear to be transparent for a good reason.  

Note that `blendWithColor:alpha:` takes an alpha argument, but that controls the blending behavior, not the opacity of the result.

Reply:
Oh, you're right, my mistake.